MobaLiveCD.txt

```diff
@@ -0,0 +1,21 @@
+[Section]
+ Name = MobaLiveCD 
+Version = 2.1
+License = GPL
+
+ Description = MobaLiveCD is a freeware that will run your Linux LiveCD on Windows thanks to the excellent emulator called "Qemu".
+ 
+Size = 1.52MB
+ 
+Category = 12
+URLSite = https://www.mobatek.net/labs_mobalivecd.html
+URLDownload = https://www.mobatek.net/exes/MobaLiveCD_v2.1.exe
+CDPath = none
+ 
+ 
+
+Size Bytes= 1.601.024 bytes
```

Comment:
Typo in parameter name.

Reply:
And add a space between the parameter name and the "equals" sign.